crates/test_utils/src/newman_runner.rs

```diff
@@ -208,3 +228,76 @@ pub fn generate_newman_command() -> ReturnArgs {
         collection_path,
     }
 }
+
+// If the connector name exists in refactorable_connector_names,
+// the corresponding collection is refactored in run time
+pub fn connector_validate(connector_name: &str) -> &str {
+    let refactorable_connector_names = ["nmi", "powertranz"];
+
+    if refactorable_connector_names.contains(&connector_name) {
+        return refactor_collection(connector_name).unwrap_or(connector_name);
+    }
+
+    connector_name
+}
```

Comment:
"Refactor" and "validate" are not the best words to be used here. Can we find something better, more specific to what we're doing?